### PR TITLE
W-11697864: test json schema nested ref by id

### PR DESCRIPTION
**Pull request by arielmirra**

### CODE REVIEW

File: amf-cli/shared/src/test/resources/validations/raml/uri-params/missing-definition.raml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#%RAML 1.0
+title: GitHub API
```
Comment (tomsfernandez) [inline amf-cli/shared/src/test/resources/validations/raml/uri-params/missing-definition.raml]: What does this have to do with the PR ?

File: amf-cli/shared/src/test/resources/validations/raml/nested-json-schema-ref/api.raml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#%RAML 1.0
+title: sMaximoLegacy
```
Comment (tomsfernandez) [inline amf-cli/shared/src/test/resources/validations/raml/nested-json-schema-ref/api.raml]: Please change API and related schema contents

**Review by tomsfernandez — CHANGES_REQUESTED** [review]

File: amf-cli/shared/src/test/resources/validations/raml/uri-params/missing-definition.raml (hunk 1 above)
Comment (tomsfernandez) [reply amf-cli/shared/src/test/resources/validations/raml/uri-params/missing-definition.raml]: Outdated, resolving.

File: amf-cli/shared/src/test/resources/validations/raml/nested-json-schema-ref/api.raml (hunk 2 above)
Comment (arielmirra) [reply amf-cli/shared/src/test/resources/validations/raml/nested-json-schema-ref/api.raml]: true, done 👌🏽

**Review by tomsfernandez — APPROVED** [review]
